djangoplicity/utils/templatetags/djangoplicity_text_utils.py

```diff
@@ -157,5 +157,5 @@ def remove_html_except(value, arg):
     for tag in soup.findAll(True):
         if tag.name not in valid_tags:
             tag.hidden = True
-        tag.attrs = [(attr, val) for attr, val in tag.attrs if attr in valid_attrs]
+        tag.attrs = [(attr, val) for attr, val in tag.attrs.items() if attr in valid_attrs]
```

Comment:
This is not the whole change, we should use dict comprehension because `attrs` in BS4 is a dict.

```suggestion
        tag.attrs = {attr: val for attr, val in tag.attrs.items() if attr in valid_attrs}
```